### PR TITLE
Added resources for (Tensorflow for deep learning)

### PR DESCRIPTION
**Pull request by DevyDhanish**
## Changes proposed
1 - Added a section for Tensorflow for Deeplearning under youtube

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Hey, looks good to me

**Review by ujjawaltyagii — COMMENTED** [review]
Looks good to me 👍

File: database/youtube/machine-learning.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+        "name":"Udacity",
+        "description":"Free online course for learning tensorflow for Deep learning",
+        "url":"https://learn.udacity.com/courses/ud187",
+        "category":"online platform",
+        "subcategory":"tensorflow",
+        "language":"english"
+    },
+    {
+        "name": "3Blue1Brown",
+        "description": "3Blue1Brown, by Grant Sanderson, is some combination of math and entertainment, depending on your disposition. The goal is for explanations to be driven by animations and for difficult problems to be made simple with changes in perspective.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:12]:
```suggestion
        "description": "3Blue1Brown, by Grant Sanderson, is a combination of math and entertainment, depending on your disposition. The goal is for difficult problems to be made simple with changes in perspective.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @DevyDhanish! 😄 I left a suggestion on how to improve your PR. Overall, well done! 👍🏾

File: database/youtube/machine-learning.json (hunk 1 above)
Comment (DevyDhanish) [reply database/youtube/machine-learning.json:12]: Thanks for review, I will make those changes.

**Comment by DevyDhanish** [comment]
Hello @CBID2 , I have made your suggested changes and commited them please review them and let me know if it need some other changes as well. Thanks.

**Review by rupali-codes — COMMENTED** [review]
lgtm

**Pull request merged**